project/sites/interchangeni/themes/interchangeni_theme/interchangeni_theme.theme

```diff
@@ -16,3 +17,30 @@ function interchangeni_theme_preprocess_node(array &$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ *
+ * New content type based suggestions to work with the opportunities titles.
+ */
+function interchangeni_theme_theme_suggestions_page_title_alter(array &$suggestions, array $variables) {
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if (!empty($node) && is_object($node)) {
+    $suggestions[] = 'page_title__node_' . $node->getType();
+  }
+}
+
+/**
+ * Implements hook_preprocess_page_title().
+ *
+ * Set a variable for use in page-title--node-opportunity.html.twig.
+ */
+function interchangeni_theme_preprocess_page_title(&$variables) {
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if ($node instanceof NodeInterface && $node->getType() === 'opportunity') {
+    $organisation = $node->get('field_manager_organisation')->getValue();
+    $organisation_nid = $organisation[0]['target_id'];
+    $organisation_name = Node::load($organisation_nid)->getTitle();
```

Comment:
You can perhaps speed up this lookup task with the `referencedEntities()` function. See https://github.com/search?q=repo%3Adof-dss%2Fnidirect-site-modules%20referencedEntities&type=code for examples